trappy/ftrace.py

```diff
@@ -445,6 +445,12 @@ def __parse_trace_file(self, trace_file):
                 continue
 
             unique_word = trace_class.unique_word
+            if unique_word in cls_for_unique_word:
```

Comment:
Admittedly, this PR should fix the bug - can this be verified in tests rather than the actual ftrace parsing code ? This seems more logical to me, but again, personal opinion.

Reply:
Yeah actually you're right, tests are the place to do this. There may have been a reason why I didn't think that was possible.. will investigate..

Reply:
Hmm, actually the reason I did this as runtime sanity-checks instead of tests was because this is a transient condition - you can't directly detect it in a test because it arises and then disappears during the  constructor of the trace object. I did add a related test for the dynamic parser overwriting a custom one, but if that failed I don't think it will immediately be obvious why.. so this RuntimeError should help.. 

I can still remove it if you prefer though, given that I Have Fixed The Bug Now™